spec/factories/hyrax_work.rb

```diff
@@ -46,6 +47,8 @@
       work.permission_manager.read_users  = evaluator.read_users
 
       work.permission_manager.acl.save
+
+      Hyrax.index_adapter.save(resource: work) if evaluator.with_index
```

Comment:
_The following comment is to document differences in processing of membership for collections and works..._

In `:hyrax_collection`, if `members` are passed in, the `after(:create)` loops through members to save and index each.  This is because the relationship is stored on the members.  

This is not required for works because the relationship is stored on the parent work, meaning there are no changes to the members when setting the relationship.